react/react1/week2/my-app/src/components/TodoList.js

```diff
@@ -0,0 +1,68 @@
+import { useState } from "react";
+import todoData from "../data/TodoData";
+
+
+export default function TodoList() {
+    const [todoState, setTodoState] = useState(todoData);
+
+    const AddNewTodo = () => {
+        setTodoState((prevTodos) => {
+            const newTodo = {
+                id: prevTodos[prevTodos.length - 1].id + 1,
+                description: "Random text",
+                status: false,
+            };
+            return [...prevTodos, newTodo];
+        });
+    };
+
+    function TaskDescription(props) {
```

Comment:
Better to create a separate file for this component, even if it's only this small

Reply:
Thank you for review!